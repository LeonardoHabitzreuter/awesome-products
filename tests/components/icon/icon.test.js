import React from 'react'
import { shallow } from 'enzyme'
import Icon from '../../../src/components/icon'

describe('Icon component', () => {
  const wrapper = shallow(<Icon type='ant-design' className='iconTest' />)

  test('should renders properly', () => {
    expect(wrapper).toMatchSnapshot()
  })

  test('should has the same type and class passed through the props', () => {
    expect(wrapper.find('Icon').props()).toMatchObject({
      type: 'ant-design',
      className: 'iconTest'
    })
  })
})
