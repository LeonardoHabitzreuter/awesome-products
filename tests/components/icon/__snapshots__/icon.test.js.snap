// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component should renders properly 1`] = `
<Icon
  className="iconTest"
  type="ant-design"
/>
`;
